### PR TITLE
Mark Fortran constants as deprecated

### LINKED ISSUE #192: Deprecated Constants (Fortran)
## Issue

Fortran constants are not marked as deprecated, e.g.:

```
C/C++/Fortran: SHMEM_REDUCE_SYNC_SIZE
```

### PR DESCRIPTION
**Pull request by jdinan**
Closes #192

### CODE REVIEW

File: utils/defs.tex [hunk 1]
```diff
@@ -84,8 +84,8 @@
 \newcommand{\EnvVarDecl}[1]{\EnvVarRef{#1}}
 \newcommand{\FuncDecl}[1]{{\ListingsCurrentStyle{#1}}\FuncIndex{#1}}
 \newcommand{\FuncParam}[1]{<{\ListingsKeywordStyle{#1}}>}
-\newcommand{\LibConstDecl}[2][\CorCppFor]{%
-  \parbox[t]{0pt}{~\\[-4pt] #1: \\\hspace*{8mm} \LibConstRef{#2} \\~}}
+\newcommand{\LibConstDecl}[2][\CorCpp]{%
+  \parbox[t]{5cm}{~\\[-4pt] #1: \\\hspace*{8mm} \LibConstRef{#2} \\~}}
 \newcommand{\LibHandleDecl}[2][\CorCppFor]{%
```
Comment (nspark) [inline utils/defs.tex]: @jdinan Can you change the default for `\LibHandleDecl` to be `CorCpp` and remove [the explicit `CorCpp` in `shmem_ctx_t`](https://github.com/openshmem-org/specification/blob/bc79f9a13a00452a8d4c87683d59213797722631/content/library_handles.tex#L16)?

Comment (jdinan) [reply utils/defs.tex]: Done -- 660e2af

**Comment by nspark** [comment]
👍

**Comment by manjugv** [comment]
👍

**Comment by jdinan** [comment]
Thanks for the reviews -- merging this as a document edit.

**Pull request merged**